src/main/java/WebRTCConf.java

```diff
@@ -0,0 +1,244 @@
+        options.setCapability("browser", "Edge");
+        options.setCapability("browser_version", "latest");
+        options.setCapability("os", "Windows");
+        options.setCapability("os_version", "10");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Edge-Safari");
+        options.setCapability("name", "WebRTC Room Creation - Edge Browser");
+        options.setCapability("browserstack.idleTimeout", 300);
+
+        return options;
+    }
+
+    public static ChromeOptions getSafariConfiguration() {
+
+        SafariOptions safariOptions = new SafariOptions();
+        safariOptions.setUseTechnologyPreview(true);
+
+        ChromeOptions options = new ChromeOptions();
+        options.merge(safariOptions);
+
+
+        options.setCapability("browser", "Safari");
+        options.setCapability("browser_version", "latest");
+        options.setCapability("os", "OS X");
+        options.setCapability("os_version", "Big Sur");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Edge-Safari");
+        options.setCapability("name", "WebRTC Room Joining - Safari Browser");
+        options.setCapability("browserstack.idleTimeout", 300);
+
+        return options;
+    }
+    
+    public static ChromeOptions getiOSConfiguration() {
+
+        SafariOptions safariOptions = new SafariOptions();
+        safariOptions.setUseTechnologyPreview(true);
+
+        ChromeOptions options = new ChromeOptions();
+        options.merge(safariOptions);
+
+
+        options.setCapability("device", "iPhone 12");
+        options.setCapability("real_mobile", "true");
+        options.setCapability("os_version", "14");
+        options.setCapability("autoAcceptAlerts", "true");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Android-iOS");
+        options.setCapability("name", "WebRTC Room Joining - iOS");
+        options.setCapability("browserstack.idleTimeout", 300);
+
+        return options;
+    }
+    
+    public static ChromeOptions getAndroidConfiguration() {
+
+    	ChromeOptions options = new ChromeOptions();
+
+        options.addArguments(
+              "--use-fake-device-for-media-stream",
+                "--use-fake-ui-for-media-stream"
+        );
+
+        options.setCapability("os_version", "11.0");
+        options.setCapability("device", "Samsung Galaxy S21");
+        options.setCapability("real_mobile", "true");
+        options.setCapability("autoGrantPermissions", "true");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Android-iOS");
+        options.setCapability("name", "WebRTC Room Creation - Android");
+        options.setCapability("browserstack.idleTimeout", 300);
+        
+        return options;
+    }
+   
+   public static void main(String[] args) {
+	   	//Disabling Selenium messages.
+	   	java.util.logging.Logger.getLogger("org.openqa.selenium").setLevel(Level.OFF);
```

Comment:
Is this a must have?

Reply:
This is added to disable Selenium INFO logs on terminal

![image](https://user-images.githubusercontent.com/83127849/141095823-7cab583c-d174-466f-844d-257ad8d831b7.png)